bool active_only
---
int8 num_drones

# size num_active_drones
float64[] x
float64[] y
float64[] z

float64[] roll
float64[] pitch
float64[] yaw